### ISSUE TITLE
Voodoo Carnival - PuPlayer issues / UltraDMD

### ISSUE THREAD

**Issue by francisdb** [jsm174/vpx-standalone-scripts#2]
These lines fail with `VBSE_ACTION_NOT_SUPPORTED`

```vbs
PuPlayer.LabelSet pBackglass,"voodoostomp",FormatNumber(stomptotalpointsheld(CurrentPlayer),0),0,""  'change to 1 to show total#
```

```vbs
PuPlayer.Init pBackglass,cGameName
```

```vbs
PuPlayer.SetBackground pBackglass,1
```

**Comment by jsm174** [comment]
So I looked into this. 

I stubbed a few more PuP functions, but unfortunately this table needs UltraDMD and I haven't attempted to port any of UltraDMD yet. 

FlexDMD can emulate UltraDMD so when I finally get to it, I will be using that.

If you do get the latest release, and add this `Exit Sub` to the script:

```SUB GetTime
        Exit Sub ' Time function is currently crashing
        Dim timevalue
        timevalue=Time
```

At least it will "boot":

https://github.com/jsm174/vpx-standalone-scripts/assets/1197137/50497483-8879-4e9c-9159-9ecce964a83c
